alerts/UA00001.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dateRaised": 20200929,
+  "affectedFirmware": ["all"],
+  "hardwareLimited": [],
+  "description": "When using interrupt driven inputs on STM32 based boards, if an external hardware failure occurs that generates an extremely high interrupt rate (over 500,000 interrupts per second) the flight controller can become overloaded and cease to function. This affects the following types of inputs: RPM sensor with RPM_TYPE 1 or 2, Camera feedback with a CAM_FEEDBACK_PIN, Wheel encoders with WENC_TYPE=1, liquid fuel flow monitors with BATT_MONITOR=11 or 12, PWM rangefinder with RFND type 5, 22 or 30, RSSI with RSSI_TYPE=4, BRD_RADIO_TYPE value 1, 2 or 3, SERIALx_PROTOCOL with value 16.",
```

Comment:
drop BRD_RADIO_TYPE and SERIALx_PROTOCOL

Reply:
Fixed